lib/blueprinter/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Blueprinter
-  VERSION = '1.0.0'
+  VERSION = '1.0.1'
```

Comment:
I was debating between `1.1.0` and `1.0.1`, thoughts?  There is a new feature in this release which makes me think maybe `1.1.0`, if it was just bug fixes then for sure `1.0.1`, but I don't have any strong opinions on how we version it.

Reply:
```suggestion
  VERSION = '1.1.0'
```
fair enough